resources/manifests/kubelet-pod-checkpointer-psp-cluster-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
The name of the file uses cluster-role-binding, while this is doing a role-binding